src/lib.rs

```diff
@@ -31,4 +32,5 @@ pub use rmath::dnorm;
 pub use rmath::pnorm;
 pub use rmath::qnorm;
 // pub use pgamma::*;
+pub use i1mach::*;
```

Comment:
```suggestion
pub use i1mach::rf_i1mach;
```
In line with discussion in #14